dvc/repo/plots/__init__.py

```diff
@@ -342,16 +342,22 @@ def _matches(targets, config_file, plot_id):
     return False
 
 
-def _dvcfile_relpath(dvcfile):
-    fs = dvcfile.repo.dvcfs
+def _normpath(path):
+    # TODO dvcfs.path.normopath normalizes to windows path on Windows
+    # even though other methods work as expected
```

Comment:
I see that this comment was in the code before. dvcfs is now posix-based always, so this should no longer be the case.

Reply:
It is since the #7477 - the problem still exists: https://github.com/iterative/dvc/runs/7298502413?check_suite_focus=true

Reply:
Related: #8020